data-raw/standard-format-data-prep/temperature_standard_format.Rmd

```diff
@@ -0,0 +1,662 @@
+#   theme_minimal()
+# 
+# summary(feather_river_temp_GRL$date)
+
+# FRA
+
+
+FRA_CDEC <- cdec_query(station = "FRA", dur_code = "H", sensor_num = "25", start_date = "1996-01-01")
+
+
+feather_river_temp_FRA <- FRA_CDEC %>%
+  mutate(date = as_date(datetime),
+         time = as_hms(datetime),
+         temp_degC = fahrenheit.to.celsius(parameter_value, round = 1)) %>%
+  select(date,time,temp_degC) %>%
+  group_by(date) %>%
+  summarise(mean_temp_degC = mean(temp_degC, na.rm = TRUE),
+            max_temp_degC = max(temp_degC, na.rm = TRUE)) %>% 
+  mutate(stream = "feather river", 
+         site = "FRA", 
+         source = "CDEC") 
+
+# ggplot() + 
+#   geom_line(data = feather_river_temp_FRA, aes(x = date, y = mean_temp_degC)) +
+#   theme_minimal()
+# 
+# summary(feather_river_temp_FRA$date)
+```
+
+### Combine Gage data for Feather River
+
+-   significant data gap exists
+
+```{r}
+
+feather_cdec_gages  <- bind_rows(feather_river_temp_FRA,
+                             feather_river_temp_GRL)
+
+ggplot() +
+  geom_line(data = feather_cdec_gages, aes(x = date, y = mean_temp_degC, color = site))
+
+```
+
+#### RST Environmental Data for Feather River 
+
+-   Chose to use this dataset since it is more complete
+
+-   Data can be aggregated up to high and low flows using `location`
+    column
+
+```{r}
+feather_rst <- rst_enviro %>% 
+  filter(stream == "feather river", parameter == "temperature") %>%
+  select(stream, site, subsite, date, value) %>%
+  rename(mean_temp_degC = value) %>%
+  mutate(source = "RST environmental") %>%
+  filter(mean_temp_degC > 0) %>%
+  glimpse
+
+ggplot() +
+  geom_line(data = feather_rst, aes(x = date, y = mean_temp_degC, color = subsite)) + 
+  facet_wrap(~site) +
+  theme_minimal()
+```
+
+## Mill Creek
+
+-   Mill Creek temperature data is sourced from CDEC's site `MLM` from
+    `1998-10-07` to `2022-07-13`
+
+```{r}
+# source USGS
+mill <- read_rds(here::here("data-raw", "standard-format-data-prep","temp_data", "mill_temp.rds"))
+mill_format <- mill %>%
+  mutate(stream = "mill creek",
+         site = stream, 
+         subsite = site,
+         source = "CDEC MLM") %>%
+   filter(mean_temp_degC > 0 & mean_temp_degC < 35,
+          max_temp_degC > 0 & max_temp_degC < 35) %>% 
+glimpse()
+
+summary(mill_format$date)
+
+mill_format %>% ggplot() + 
+  geom_line(aes(x = date, y = mean_temp_degC), color = "gray") + 
+  # geom_point(aes(x = date, y = mean_temp_degC), size = .001) +
+  theme_minimal()
+```
+
+## Sacramento
+
+-   combined dataset spans `1998-10-01` to `2022-07-25` and includes RST
+    environmental data from Knights Landing and USGS data from Tisdale
+
+#### Gage Data
+
+-   Data is from `USGS 11390500` and exists from `1998-10-01` to
+    `2022-07-25` with a significant data gap between `1998-09-30` and
+    `2016-10-01`
+  - use for both Knights Landing and Tisdale
+
+```{r}
+
+### USGS 11390500
+# Read created CSV
+WLK_USGS <- readNWISdv(11390500, "00010")
```

Comment:
It would be great if this could also be read in as an rds or csv to be consistent with how the rest of them function - would be quicker than a data pull.